app/src/main/java/org/gdg/frisbee/android/activity/AboutActivity.java

```diff
@@ -61,22 +55,20 @@ public void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().setDisplayUseLogoEnabled(true);
         getSupportActionBar().setTitle(R.string.about);
 
-
         mIndicator.setOnPageChangeListener(this);
 
-        mViewPagerAdapter = new AboutPagerAdapter(this, getSupportFragmentManager());
-        mViewPager.setAdapter(mViewPagerAdapter);
+        mViewPager.setAdapter(new AboutPagerAdapter(this, getSupportFragmentManager()));
         mIndicator.setViewPager(mViewPager);
     }
 
     protected String getTrackedViewName() {
-        return "About/"+getResources().getStringArray(R.array.about_tabs)[getCurrentPage()];
+        return "About/" + getResources().getStringArray(R.array.about_tabs)[getCurrentPage()];

```

Comment:
We live, we code, we learn. Even with this public code you got into Google. :tongue: jk, you did a fine job with this app but now it's time to complain about the style of each line. That's how it goes. :smirk: